scipy/stats/_continuous_distns.py

```diff
@@ -2267,7 +2267,8 @@ class f_gen(rv_continuous):
 
     for :math:`x > 0` and parameters :math:`df_1, df_2 > 0` .
 
-    `f` takes ``dfn`` and ``dfd`` as shape parameters.
+    `f` takes ``dfn`` (numerator, :math:`df_1`) and ``dfd`` (denominator, :math:`df_2`)
+    as shape parameters.
```

Comment:
Without mentioning the relationship between the `f` distribution and a ratio of chi-square random variables, the terms "numerator" and "denominator" are likely to be misunderstood as referring to the numerator and denominator of the PDF shown above. 

How about:
```suggestion
    `f` accepts shape parameters ``dfn`` and ``dfd`` for :math:`df_1`
    and :math:`df_2`, respectively.
```

Reply:
What about replacing
```

```
with
```
The f-distribution with :math`df_1` and :math:`df_2` degrees of freedom is the distribution of the ratio of two independent chi-squared distributions with :math`df_1` and :math`df_1` degrees of freedom, after rescaling by :math`df_2 / df_1`.
The probability density function for `f` is:
```

and then

```
`f` accepts shape parameters ``dfn`` and ``dfd`` for :math:`df_1`, the degrees of freedom of the chi-squared distribution in the numerator, and :math:`df_2`, the degrees of freedom of the chi-squared distribution in the denominator, respectively.
```

Reply:
Sure, you can include that. 
If we're going to do all that, what about also replacing subscripts 1 and 2 with the letters $n$ and $f$ in the math?
Then I think you can omit the "the degrees of freedom of the chi-squared distribution in the numerator" etc. in the last sentence.

Reply:
I would prefer to keep the subscripts 1 and 2, as this is what the degrees are referred to in the literature. A consistent approach would rename the arguments of `f` to `df1` and `df2`, but this would be a breaking change.

Reply:
Then please push the commit with your latest suggestion minus "the degrees of freedom of the chi-squared distribution in the numerator", etc.. in the last line.
Since the information it contains isn't new, the intent seems to be to emphasize the relationship between "n" and numerator, but it doesn't do that explicitly. So if a reader still hasn't made the connection, they still might not. 

If it's really important to make that connection, how about an explicit, separate sentence, e.g. "Note that the subscript $n$ corresponds with the chi square random variable in the *numerator* and...".

Reply:
Next commit, please include `[skip actions] [skip cirrus]` in the commit message.

Reply:
I didn't see you already pushed. Never mind; I'll just merge.

Reply:
Thanks @mdhaber.